db/controllers/api.controller.js

```diff
@@ -97,3 +98,15 @@ exports.patchArticles = (req, res, next) => {
       next(err);
     });
 };
+
+exports.deleteCommentById = (req, res, next) => {
+  const { comment_id } = req.params;
+  removeCommentById(comment_id)
+    .then(() => {
+      // console.log("Comment deleted");
```

Comment:
Don't forget to remove commented out code before you push! It's best to delete `console.logs` once you've debugged the thing you were interested in, instead of commenting it out